webknossos/webknossos/dataset/layer.py

```diff
@@ -532,16 +590,18 @@ def delete_mag(self, mag: Union[int, str, list, tuple, np.ndarray, Mag]) -> None
                 "Deleting mag {} failed. There is no mag with this name".format(mag)
             )
 
+        full_mag_path = _find_mag_path(
+            self.dataset.path, self.name, mag.to_layer_name(), self.mags[mag].path
+        )
+        assert is_fs_path(full_mag_path), "Cannot delete remote mags."
+
```

Comment:
This throws an error in the tests. It is really true, that wklibs support deleting remote mags? e.g. on s3?